applications/GeoMechanicsApplication/tests/line_load_tests/line_loads_in_stages/Orchestrator/ProjectParametersLoadFromCheckpoint.json

```
@@ -0,0 +1,155 @@
+    {
+"orchestrator" : {
+    "name" : "Orchestrators.KratosMultiphysics.SequentialOrchestrator",
+    "settings" : {
+        "echo_level" : 0,
+        "execution_list" : ["stage_2"],
+        "load_from_checkpoint" : "checkpoints/stage_1",
+        "stage_checkpoints" : true,
```

I believe we should set this to `false`, since we don't intend to save stages:
```suggestion
        "stage_checkpoints" : false,
```

---

Done